### ISSUE TITLE
Rapid Photo Downloader doesn't start

### ISSUE THREAD

**Issue by Nebucatnetzer** [NixOS/nixpkgs#208249]
### Describe the bug

When I try to start `rapid-photo-downloader` I get the following error.

```
Traceback (most recent call last):
  File "/nix/store/asw6divl0di5ikmz8g4w3z57a48mgma0-rapid-photo-downloader-0.9.18/bin/.rapid-photo-downloader-wrapped", line 6, in <module>
    from raphodo.rapid import main
  File "/nix/store/asw6divl0di5ikmz8g4w3z57a48mgma0-rapid-photo-downloader-0.9.18/lib/python3.10/site-packages/raphodo/rapid.py", line 62, in <module>
    gi.require_version('Notify', '0.7')
  File "/nix/store/k46f21qglnnl2grkx6gq6j6jlawfkgpw-python3.10-pygobject-3.42.2/lib/python3.10/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Notify not available
```

### Steps To Reproduce
1. `nix-shell -p rapid-photo-downloader --run rapid-photo-downloader`

### Expected behavior

The application opens.

### Notify maintainers

@jfrankenau 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.82, NixOS, 22.11 (Raccoon), 22.11.20221224.dac57a4`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.1`
 - channels(root): `""`
 - nixpkgs: `/nix/store/q51smsq548y5y5qrncdqvihhscxfpp47-source`
```

**Comment by marmay** [comment]
I ran into the same issue.

I've attached a patch that fixes all the issues. First, the GI_TYPELIB_PATH was not correct. I do not know, whether there is a better way to end up with a correct path, but this worked for me.

Second, due to the update to python3.10, the old rapid-photo-downloader version no longer worked. I've updated it and included new dependencies.

[update_rapid.txt](https://github.com/NixOS/nixpkgs/files/10335227/update_rapid.txt)
